src/pages/Main/Main.js

```diff
@@ -1,12 +1,83 @@
-import React from 'react';
-import Nav from '../../components/Nav.js/Nav';
+import { React, useCallback, useEffect, useState } from 'react';
```

Comment:
```suggestion
import React, {useEffect, useState } from 'react';
```
React는 default이고, useCallback사용하지 않으면 삭제해주세요!

Reply:
> 은지님 리뷰 반영해주세요!

넵...! 다시 푸시하겠습니다